docs/autolinking.md

```diff
@@ -1,20 +1,14 @@
 # Autolinking
 
-Autolinking is a mechanism built into CLI that allows adding a dependency with native components for React Native to be as simple as:
+React Native libraries usually come with platform-specific (native) code. Autolinking is a mechanism that allows your project to discover and use this code.
 
-```sh
-yarn add react-native-webview
-```
+Add a library using your favorite package manager and run the build. That's it. No more editing build config files to use native code.
```

Comment:
This sounds confusing. Can you specify which package managers you are referring to? An example is helpful too.

Reply:
Yea, how about that: fb156f4? (funny hash)